test/seq/runtests.jl

```diff
@@ -782,6 +782,54 @@ end
         @test convert(DNASequence, RNASequence("ACGUN")) == dna"ACGTN"
     end
 
+    @testset "Conversion to Matrices" begin
+        dna = [dna"AAA", dna"TTTAAA", dna"CCC", dna"GGG"]
+        rna = [rna"AAA", rna"UUU", rna"CCCUUU", rna"GGG"]
+        prot = [aa"AMG", aa"AMG", aa"AMG", aa"AMG"]
+        sitemajdna = [
+            DNA_A  DNA_A  DNA_A;
+            DNA_T  DNA_T  DNA_T;
+            DNA_C  DNA_C  DNA_C;
```

Comment:
A newline and a semicolon are interchangeable. So semicolons are redundant when you break rows with newlines.